bzira.py

```diff
@@ -187,11 +188,14 @@ def parse_options():
     
 bz = bugzilla.Bugzilla(url=bzserver + "bugs/xmlrpc.cgi")
 
-print "[DEBUG] " + "Querying bugzilla: " + query
+
+if (options.verbose):
+    print "[DEBUG] " + "Querying bugzilla: " + query.replace("last_change_time","chfieldfrom")
```

Comment:
The query you actually send is NOT buglist.cgi, it's xmlrpc.cgi. So this is an invalid URL either way. Anyway, I've reproduced this fix in #19 so you can just reject this PR and use that newer one, with all my other typo & grammar corrections, too.